### ISSUE TITLE
License terms?

### ISSUE THREAD

**Issue by Rosuav** [brillout/forge-sha256#1]
I'm needing a simple SHA256 hash function for a project I'm working on, and yours is tiny and convenient. However, I need the hash in binary form, not the hex that your wrapper returns. So I've taken a copy of the minified version and tweaked it to return base64. Unfortunately you don't have any license terms in the project, so I'm not sure if what I've done is legal; if you're not happy with me doing this, I'll rip it out and find an alternative.

Can you add a LICENSE or LICENSE.md to the project, please?

**Comment by brillout** [comment]
I'll happily merge a PR with whatever suiting License:)

**Comment by Rosuav** [comment]
Sure. I use the MIT license for most of my code; it basically says (in layman's terms) "here, have fun, just don't sue me".
